CHANGELOG.latest.md

```diff
@@ -1,4 +1,2 @@
-### Dependency Updates
-* [AUTOMATIC] iOS 4.16.0 => 4.17.2 Android 5.6.6 => 5.6.7 (#298) via RevenueCat Git Bot (@RCGitBot)
-### Other Changes
-* Update Gemfile.lock (#296) via Cesar de la Vega (@vegaro)
+### New Features
+* Added `CommonFunctionality.setLogLevel` (#297) via NachoSoto (@NachoSoto)
```

Comment:
Hmm it might have been good to mention this was only on iOS... I think we can update it on the next release

Reply:
`CommonFunctionality` is the iOS name though.

Reply:
True... Since PHC is mostly internal, that's ok 👍